locals.tf

```
@@ -5,8 +5,18 @@
 # ------------------------------------------------------------------------------
 data "aws_caller_identity" "current" {}
 
+# Retrieve information about the findings data bucket we will be providing access
+# for with this workspace.
+data "aws_s3_bucket" "findings_data" {
+  bucket = local.findings_data_s3_bucket
+}
+
 locals {
   # This is a goofy but necessary way to determine if
   # terraform.workspace contains the substring "prod"
   production_workspace = replace(terraform.workspace, "prod", "") != terraform.workspace
```

As long as we are cleaning up, let's use a regex here:
```suggestion
  # Determine if this is a Production workspace by checking
  # if terraform.workspace begins with "prod-"
  production_workspace =  length(regexall("^prod-", terraform.workspace)) == 1
```

---

Ah, much nice. Glad you noticed!